gms/gossiper.cc

```diff
@@ -1885,7 +1885,8 @@ future<> gossiper::do_shadow_round(std::unordered_set<gms::inet_address> nodes)
             gms::application_state::DC,
             gms::application_state::RACK,
             gms::application_state::SUPPORTED_FEATURES,
-            gms::application_state::SNITCH_NAME}};
+            gms::application_state::SNITCH_NAME,
+            gms::application_state::RAFT_SERVER_ID}};
```

Comment:
I suppose this will go away if we merge the raft server id and the node id?

Reply:
Yes.

Reply:
BTW, HOST_ID is published by a node unconditionally. It is also the first thing a node would announce. So if one sees a gossip entry in for a node, a HOST_ID will always present. 

Once HOST_ID is unique, we can get rid of the RAFT_SERVER_ID all together.